packages/rn-tester-e2e/tests/screens/components.screen.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+import {UtilsSingleton as Utils} from '../helpers/utils';
+
+// root level screen in RNTester: Components
+
+class ComponentsScreen {
+  // Reference in the top level Component list
+
+  buttonComponentLabelElement: string = Utils.platformSelect({
+    ios: '[label="Button Simple React Native button component."]',
+    android: '~Button Simple React Native button component.',
+  });
+
+  // Methods to interact with top level elements in the list
+
+  async checkButtonComponentIsDisplayed(): Promise<boolean> {
+    return await Utils.checkElementExistence(this.buttonComponentLabelElement);
+  }
+
+  async clickButtonComponent(): any {
```

Comment:
```suggestion
  async clickButtonComponent(): void {
```

Reply:
Same for all of the `Promise<any>` examples below.

Reply:
Hi @kelset, 
This is fixed in the last MR. Could you please resolve this comment?